redisinsight/ui/src/components/virtual-grid/VirtualGrid.tsx

```diff
@@ -0,0 +1,256 @@
+
+    if (rowIndex === 0) {
+      return (
+        <div style={style}>
+          <div className={cx(styles.gridHeaderItem, 'truncateText')}>
+            {isObject(content) && (
+              <>
+                {!!content?.sortable && (
+                  <button
+                    type="button"
+                    data-testid="header-sorting-button"
+                    className={styles.gridHeaderItemSortable}
+                    onClick={() => changeSorting(column.id)}
+                  >
+                    <span>{content.label}</span>
+                    <span style={{ paddingLeft: 0 }}>
+                      <EuiIcon
+                        style={{ marginLeft: '4px' }}
+                        type={sortedColumn?.order === SortOrder.DESC ? 'sortDown' : 'sortUp'}
+                      />
+                    </span>
+                  </button>
+                )}
+                {!content?.sortable && (<div>{content.label}</div>)}
+              </>
+            )}
+            {!isObject(content) && content}
+            {!isObject(content) && !content && <div>&nbsp;</div> }
+          </div>
+        </div>
+      )
+    }
+    if (columnIndex === 0) {
+      const lastColumn = columns[columns.length - 1]
+      return (
+        <div
+          className={cx(styles.gridItem,
+            rowIndex % 2
+              ? styles.gridItemOdd
+              : styles.gridItemEven)}
+          style={style}
+        >
+          {column?.render && isObject(rowData) && column?.render(rowData) }
+          {!column?.render && content }
+
+          <div
+            className={cx(styles.gridItem, styles.gridItemLast,
+              rowIndex % 2
+                ? styles.gridItemOdd
+                : styles.gridItemEven)}
+            style={{
+              width: lastColumn?.minWidth,
+              height: rowHeight,
+              marginLeft: width - lastColumn?.minWidth - 29
+            }}
+          >
+            {lastColumn?.render && isObject(rowData) && lastColumn?.render(rowData) }
+          </div>
+        </div>
+      )
+    }
+
+    return (
+      <div
+        className={cx(styles.gridItem,
+          rowIndex % 2
+            ? styles.gridItemOdd
+            : styles.gridItemEven)}
+        style={style}
+      >
+        {column?.render && isObject(rowData) && column?.render(rowData) }
+        {!column?.render && content }
+      </div>
+    )
+  }
+
+  const innerElementType = useInnerElementType(
+    Cell,
+    columnWidth,
+    () => rowHeight,
+    columns.length - 1,
+  )
+
+  return (
+    <div
+      className={styles.container}
+      onWheel={onWheel}
+      data-testid="virtual-grid-container"
+    >
+      {loading && !hideProgress ? (
+        <EuiProgress
+          color="primary"
+          size="xs"
+          position="absolute"
+          className={styles.progress}
+          data-testid="progress-entry-list"
+        />
+      ) : null}
```

Comment:
it will be enough to do
`{(loading && !hideProgress) && (<></>)}`

Reply:
changed